core/src/main/java/com/klaytn/caver/transaction/type/ValueTransfer.java

```diff
@@ -0,0 +1,269 @@
+package com.klaytn.caver.transaction.type;
+
+import com.klaytn.caver.Klay;
+import com.klaytn.caver.crypto.KlaySignatureData;
+import com.klaytn.caver.transaction.AbstractTransaction;
+import com.klaytn.caver.utils.BytesUtils;
+import com.klaytn.caver.utils.Utils;
+import org.web3j.rlp.*;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Represents a value transfer transaction.
+ * Please refer to https://docs.klaytn.com/klaytn/design/transactions/basic#txtypevaluetransfer to see more detail.
+ */
+public class ValueTransfer extends AbstractTransaction {
+    /**
+     * The account address that will receive the transferred value.
+     */
+    String to;
+
+    /**
+     * The amount of KLAY in peb to be transferred.
+     */
+    String value = "0x00";
```

Comment:
I think for value should be defined by user with ValueTransfer transaction.
How about define value without default value?